UT4MasterServer.Services/Scoped/RatingsService.cs

```diff
@@ -119,20 +121,35 @@ public async Task<PagedResponse<RankingsResponse>> GetRankingsAsync(string ratin
 		var accountIds = ratings.Select(s => s.AccountID);
 		var filterAccounts = Builders<Account>.Filter.In(f => f.ID, accountIds);
 		var fields = Builders<Account>.Projection.Include(i => i.Username);
```

Comment:
`fields` is not used
```suggestion
		var filterAccounts = Builders<Account>.Filter.In(f => f.ID, accountIds);
```

Reply:
Removed it.